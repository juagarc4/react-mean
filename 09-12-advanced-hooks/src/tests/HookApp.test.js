import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import { HookApp } from 'HookApp'

const wrapper = shallow(<HookApp />)

describe('Tests in <HookApp />', () => {
  test('Should show the component correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
