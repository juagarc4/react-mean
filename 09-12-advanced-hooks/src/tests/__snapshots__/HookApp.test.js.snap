// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HookApp /> Should show the component correctly 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
